src/core/utils/DtoMapping.ts

```diff
@@ -76,6 +77,22 @@ export class DtoMapping {
         );
     }
 
+    /**
+     * This method knows how to convert an address payload sent by Rest to Address.
+     *
+     * Currently rest sends hex encoded addresses, it is desired to use base32 encoded addresses.
+     *
+     * This method handles both format, encoded (deprecated) and base32 encoded addresses.
+     *
+     * Clients using this SDK will be able to process both payloads.
+     *
+     * @param value the address in encoded (6823BB7C3C089D996585466380EDBDC19D4959184893E38C) format or decoded/plain format (SB3KUBHATFCPV7UZQLWAQ2EUR6SIHBSBEOEDDDF3)
+     * @return the Address object.
+     */
+    public static toAddress(value: string): Address {
+        return Convert.isHexString(value, 48) ? Address.createFromEncoded(value) : Address.createFromRawAddress(value);
```

Comment:
If the recipientAddress is alias `THBIMC3THGH5RUYAAAAAAAAAAAAAAAAAAAAAAAA`

It will throw an error `Address Network unsupported` right?

Reply:
We haven't removed private/mijin network support from the old SDKs yet. Unsure if it makes sense to remove them, this SDK would fade off eventually.

Reply:
Ignore my question. 
Actually, I misunderstood, even though the alias namespace is still converted to a valid address format from REST.